### ISSUE TITLE
CSS Not Appropriate for Injected Page Layout Code

### ISSUE THREAD

**Issue by jacobjohnston** [Azure-Samples/Azure-AD-B2C-page-templates#14]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [x] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> 1.) Use this template for the latest Page Layout version (2.1.0)
> 2.) Observe that many elements are not styled appropriately.

### Any log messages given by the failure
> N/A

### Expected/desired behavior
> The CSS from this repository to render all elements the same way using the built-in templates do.

### OS and Version?
> N/A

### Versions
> Azure Page Layout 

### Mention any other details that might be useful
I've found that a number of the element IDs have changed in a way where they no longer to work with this template. For example, `#email_ver_but_send now` appears to be `#emailVerificationControl_but_send_code`, `#logonIdentifier` or `#signInName` now appears to be , `#email`. This repository should be updated when the page layout versions change.

**Comment by ckjacksoncda** [comment]
Yep!  I can also confirm this.  Cost me a few handfuls of hair.  lol

**Comment by cmaneiro** [comment]
I'm seeing the same issue in the unified page, I have solve it adding **.unified_container .row .panel-default #api .localAccount .entry .entry-item input** to the unified.css file Line 109.
